twisted/conch/ssh/_kex.py

```diff
@@ -153,6 +198,19 @@ def getKex(kexAlgorithm):
     return _kexAlgorithms[kexAlgorithm]
 
 
+def isEllipticCurve(kexAlgorithm):
```

Comment:
If it's a matter of taste then I'll keep it as is.  It seems to fit the convention used by isFixedGroup and in my eyes easier to understand.